R/FilterState.R

```diff
@@ -76,6 +76,7 @@ FilterState <- R6::R6Class( # nolint
                           varlabel = character(0),
```

Comment:
we should make sure the default x_reactive works as I'd love to keep being able to use filterstate outside of filterstates as it makes it easier to understand the classes

Reply:
Done - I've added default value `reactive(NULL)` to all classes